LBH.AdultSocialCare.Api/Startup.cs

```diff
@@ -89,7 +89,6 @@ public void ConfigureServices(IServiceCollection services)
 
             services.ConfigureDbContext(Configuration);
             services.AddAuthentication();
-            services.ConfigureIdentityService();
```

Comment:
No possible use for this configuration? Unique email maybe?

Reply:
oops... removed it by mistake, when experimenting with authentication mocks
Now is restored, thank you